hw/top_earlgrey/data/ip/chip_uart_testplan.hjson

```diff
@@ -74,5 +74,81 @@
       lc_states: ["RMA"]
       tests: ["chip_sw_uart_tx_rx_alt_clk_freq", "chip_sw_uart_tx_rx_alt_clk_freq_low_speed"]
     }
+    {
+      name: chip_sw_uart_parity
+      desc: '''Verify transmission of data in both directions with parity enabled.
+
+            This is very similar to chip_sw_uart_tx_rx except that parity (generation and checking)
+            should be enabled. Control expected word parity by configuring CTRL.PARITY_ODD register.
```

Comment:
Nit: I think we usually do one sentence per line instead of line wrapping in HJSON, but this is not a huge deal

Reply:
I'm not sure that there's a proper convention here: I just looked at the other files in the directory and I think I can find both systems :-) So I'm going to keep with the version that's more convenient to edit, but we can always tweak it later.

Reply:
Ok, that sounds fine to me.